pkg/lib/oauth/handler/handler_token.go

```diff
@@ -789,14 +808,7 @@ func (h *TokenHandler) issueTokensForAuthorizationCode(
 		SSOEnabled:         code.SSOEnabled,
 	}
 	if issueRefreshToken {
-		// First revoke existing refresh tokens if MaxConcurrentSession == 1
-		if client.MaxConcurrentSession == 1 {
-			err := h.revokeClientOfflineGrants(client, code.AuthenticationInfo.UserID)
-			if err != nil {
-				return nil, err
-			}
-		}
-		offlineGrant, err := h.TokenService.IssueOfflineGrant(client, opts, resp)
+		offlineGrant, err := h.issueOfflineGrant(client, code, resp, opts)
```

Comment:
In the `TokenHandler`, we have multiple places that call `h.TokenService.IssueOfflineGrant` to issue the refresh token. Should we also update those places to use `h.issueOfflineGrant` instead? (e.g. `handleBiometricAuthenticate`, to restrict biometric login concurrent sessions ...etc.) If yes, please check the functionality after updating

Reply:
I guess yes, let me check it